### DISCUSSION TITLE
Hardware Buttons w/o action on Relays | Hardware Buttons steuern Relais nicht an

### DISCUSSION THREAD

**Discussion by breitgo** [Blackymas/NSPanel_HA_Blueprint#2192] [Q&A] [answered]
Dear all,

I assume this is a very, very dumb question - but still, I cannot figure out how to get it working. 
Did the installation via ESPHome and the @Blackymas blueprint. My problem is that the hardware buttons are not working and I cant find a solution on how to activate them. Via HA I can "switch" them on / off and you hear the relais clicking. What do I need to do to make them work physically by touching/switching/operating them directly from the panel (e.g. to power on / off an un-smart fan). 
Thanks for your help!
Christian

**Comment by edwardtfn** [comment, marked as answer]
On the Blueprint settings, assign the hardware buttons to its respective physical relay entity and it shoukd work...
